lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -44,7 +44,17 @@ def authorize(money, payment, options = {}, capture = false)
         post = build_auth_request(money, payment, options)
         post[:processingInformation] = { capture: true } if capture
 
-        commit('/pts/v2/payments/', post)
+        commit('payments/', post)
+      end
+
+      def refund(money, authorization, options = {})
+        post = build_refund_request(money, options)
+        commit("payments/#{authorization}/refunds", post)
+      end
+
+      def credit(money, payment, options = {})
+        post = build_credit_request(money, payment, options)
+        commit('credits', post)
```

Comment:
Should this be `credits/`?

Reply:
No. Thanks, I removed  `/` in the `payments` method